src/System.IO.FileSystem.AccessControl/tests/DirectoryObjectSecurityTests.cs

```diff
@@ -493,6 +493,23 @@ public void RemoveAccessRuleAll_AccessControlType_Allow_Succeeds()
             Assert.False(existingRules.Contains(customAccessRuleSynchronize));
         }
 
+        [Fact]        
+        public void RemoveAccessRuleAll_AccessControlType_Deny_ThrowException()
```

Comment:
@pjanotti i've found one test where assertion fail.

Reply:
@MarcoRossignoli hi Marco, sorry for the long delay, I was on vacation...

Great that you found this case, thanks for keep digging on it!